app/src/main/java/com/rockthevote/grommet/util/DispatcherProvider.kt

```diff
@@ -0,0 +1,12 @@
+package com.rockthevote.grommet.util
+
+import kotlinx.coroutines.CoroutineDispatcher
+
+/**
+ * Provides Coroutine dispatchers for better control within the application and tests
+ */
+interface DispatcherProvider {
```

Comment:
didn't we already have  a DispatcherProvider interface?

Reply:
ah, you moved it from the coroutines folder, ya?

Reply:
I swore we did, but I can't find it.

Reply:
You know what, it was in the other PR I think. I see it now, I'm removing this.

Reply:
👍